pypeclub/OpenPype
# Deadline: priority configurable in Maya jobs

## Brief description
Priority is pulled from Settings when instance is being created and it is propagated into Deadline.

## Description
Added separate `tile_priority` to differentiate regular render and tile jobs.

![maya_dl_priority](https://user-images.githubusercontent.com/4457962/161227122-fa1e8f10-cf52-40c9-9228-de72470e4eb0.png)

## Testing notes:
1. modify `project_settings/deadline/publish/MayaSubmitDeadline` priorities
2. create instances of 'render' family in Maya
3. priorities should be visible when clicked on instance in a list of user configurable attributes
4. submit to Deadline, monitor priorities (accessible in `Job Priorities` (double click on job name), section `General`)
